website/settings/defaults.py

```diff
@@ -79,11 +87,15 @@ def parent_dir(path):
 SHARE_ELASTIC_INDEX_TEMPLATE = 'share_v{}'
 
 # Sessions
-# TODO: Override OSF_COOKIE_DOMAIN in local.py in production
-OSF_COOKIE_DOMAIN = None
 COOKIE_NAME = 'osf'
-# TODO: Override SECRET_KEY in local.py in production
+OSF_COOKIE_DOMAIN = None
```

Comment:
The removed TODO items served as deployment notes- was there any particular reason for removing them?